src/app/3d/qgspoint3dsymbolwidget.cpp

```diff
@@ -218,9 +222,12 @@ QgsAbstract3DSymbol *QgsPoint3DSymbolWidget::symbol()
       break;
   }
 
-  const QQuaternion rot( QQuaternion::fromEulerAngles( spinRX->value(), spinRY->value(), spinRZ->value() ) );
-  const QVector3D sca( spinSX->value(), spinSY->value(), spinSZ->value() );
-  const QVector3D tra( spinTX->value(), spinTY->value(), spinTZ->value() );
+  // A point on the 2D plane (x', y') is transformed to (x, -z) in the 3D world.
+  // The rotation, scale and translation values need to be converted in the 3D world.
+  QgsVector3D translationWorldCoords = Qgs3DUtils::mapToWorldCoordinates( QgsVector3D( spinTX->value(), spinTY->value(), spinTZ->value() ), QgsVector3D( 0, 0, 0 ) );
```

Comment:
Could you please remove the use of Qgs3DUtils::mapToWorldCoordinates() with "fake" origin from here as well?

Reply:
Done.